examples/3d-tiles/app.js

```diff
@@ -5,10 +5,16 @@ import DeckGL from '@deck.gl/react';
 import {COORDINATE_SYSTEM, OrbitView, LinearInterpolator} from '@deck.gl/core';
 import {PointCloudLayer} from '@deck.gl/layers';
 
-import {Tile3DLoader, Tile3DFeatureTable, Tile3DBatchTable} from '@loaders.gl/3d-tiles';
+import {
+  Tile3DLoader,
+  Tile3DFeatureTable,
+  Tile3DBatchTable,
+  parseRGB565
```

Comment:
Perhaps this could be an option to the Tile3DLoader (i.e. whether to convert 565 colors to RGB colors)?

PS - WebGL can actually render RBG_565 colors directly but we might need some change to the point cloud layer or attribute manager, so having the conversion option doesn't hurt.